README.md

```diff
@@ -0,0 +1,20 @@
+#LaunchPad Quick Start for Android
+
+The LaunchPad project is intended to be an aid for helping you bring your Android projects to market quickly without the need to reinvent the wheel.  It is open source and free to use.
+
+## LaunchPad Compose
+This library provides a collection of Compose widgets and utility functions with the goal of making coding in Compose that much easier.
+
+## Contributing
+This project is open source and we welcome updates from all devs.  Look to [`CONTRIBUTING.md`](./CONTRIBUTING.md) for information on how to help out.
+
+## What's in this library
+
+### Convenience Functions
+
+* Check out ThemeExtensions.kt for a collection of easy to use functions for making your Compose Themes easier to read.
```

Comment:
@BottleRocket-Colin nit: Consider either:

1. Linking to the Github file link - [ThemeExtensions.kt](https://github.com/BottleRocketStudios/Android-LaunchPad-Compose/blob/main/launchpad-compose/src/main/java/com/bottlerocketstudios/launchpad/compose/ThemeExtensions.kt) OR
2. Use the inline code style around the filename like `ThemeExtensions.kt`